editor/src/clj/editor/model_scene.clj

```diff
@@ -323,7 +323,10 @@
   (:animation-ids content))
 
 (g/defnk produce-material-ids [content]
-  (:material-ids content))
+  (let [ret (:material-ids content)]
+    (if (zero? (count ret))
```

Comment:
The Collada model loader currently reports 0 materials, but we really need to have at least one.